packages/share_plus/share_plus/pubspec.yaml

```diff
@@ -37,12 +37,12 @@ dependencies:
     sdk: flutter
   share_plus_platform_interface: ^5.0.0
   file: ">=6.1.4 <8.0.0"
-  url_launcher_web: ^2.0.16
-  url_launcher_windows: ^3.0.6
-  url_launcher_linux: ^3.0.5
-  url_launcher_platform_interface: ^2.1.2
-  ffi: ^2.0.1
-  web: ^0.5.0
+  url_launcher_web: ^2.3.2
```

Comment:
We only need to update this part of `url_launcher` to move to `web: 1.0.0`, but updated the rest as well.